tests/unit/source_code/conftest.py

```diff
@@ -55,3 +58,8 @@ def simple_dependency_resolver(mock_path_lookup: PathLookup) -> DependencyResolv
     notebook_resolver = NotebookResolver(NotebookLoader())
     import_resolver = ImportFileResolver(FileLoader(), allow_list)
     return DependencyResolver(library_resolver, notebook_resolver, import_resolver, import_resolver, mock_path_lookup)
+
+
+@pytest.fixture
+def mock_dfsa_crawlers() -> DirectFsAccessCrawlers:
```

Comment:
use `create_autospec(DirectFsAccessCrawlers)` in every test that uses it

Reply:
done